packages/ui-library/src/components/forms/textarea/index.ts

```diff
@@ -154,14 +154,16 @@ export class BlrTextarea extends LitElement {
           ${dynamicStyles}
         </style>
         <div class="${classes} blr-textarea">
-          ${BlrFormLabelRenderFunction({
-            labelText: this.label,
-            labelSize: this.size,
-            labelAppendix: this.labelAppendix,
-            forValue: this.textareaId,
-            theme: this.theme,
-            variant: this.hasError ? 'error' : 'label',
-          })}
+          <div class="label-wrapper">
+            ${BlrFormLabelRenderFunction({
```

Comment:
Is this intentional that label presence is not checked for textarea but is checked for text-input? Right now the hasLabel property is a dead code as it is not used anywhere in this file except during declaration.

Reply:
Ok, I see what you mean. I'll add in a check for hasLabel and include label-wrapper within that check. I can apply this to all components that use it